runners/core-java/src/test/java/org/apache/beam/runners/core/ReduceFnTester.java

```diff
@@ -173,9 +173,8 @@
           throws Exception {
 
     CoderRegistry registry = CoderRegistry.createDefault();
-    AppliedCombineFn<String, Integer, AccumT, OutputT> fn =
-        AppliedCombineFn.withInputCoder(
-            combineFn, registry, KvCoder.of(StringUtf8Coder.of(), VarIntCoder.of()));
+    AppliedCombineFn.withInputCoder(
```

Comment:
Add a comment that we are ensuring that we can convert the CombineFn into an AppliedCombineFn otherwise the reason for this statement is not obvious.